homeassistant/components/modem_callerid/sensor.py

```diff
@@ -42,6 +46,9 @@ async def _async_on_hass_stop(event: Event) -> None:
     platform = entity_platform.async_get_current_platform()
 
     platform.async_register_entity_service(SERVICE_REJECT_CALL, {}, "async_reject_call")
+    _LOGGER.warning(
+        "Calling reject_call service is deprecated and will be removed after 2022.4"
```

Comment:
Can we add a little more to it? As in, when a user gets this in their logs, they will have no clue what to do.

```
Calling mode_callerid.reject_call service is deprecated and will be removed after 2022.4, please use the new reject call button entity as a replacement.
``